libvips/iofuncs/targetcustom.c

```diff
@@ -148,11 +176,27 @@ vips_target_custom_class_init( VipsTargetCustomClass *class )
 		G_TYPE_POINTER, G_TYPE_INT64 );
 
 	/**
-	 * VipsTargetCustom::finish:
+	 * VipsTargetCustom::end:
 	 * @target_custom: the target being operated on
 	 *
 	 * This signal is emitted at the end of write. The target should do
 	 * any finishing necessary.
+	 *
+	 * Returns: 0 on success, -1 on error.
+	 */
+	vips_target_custom_signals[SIG_END] = g_signal_new( "end",
+		G_TYPE_FROM_CLASS( class ),
+		G_SIGNAL_ACTION,
+		G_STRUCT_OFFSET( VipsTargetCustomClass, end ), 
+		NULL, NULL,
+		vips_INT__VOID,
+		G_TYPE_NONE, 0 );
```

Comment:
```suggestion 
		G_TYPE_INT, 0 );
```

I think that would fix this error:
```
(process:20208): GLib-GObject-CRITICAL **: 16:25:55.597: g_value_set_int: assertion 'G_VALUE_HOLDS_INT (value)' failed
```
(untested)

Reply:
Ooop crap. Thanks!!